concepts/tuples/about.md

```diff
@@ -0,0 +1,119 @@
+# Tuples
+
+A [tuple][tuple] is a finite ordered list of elements which is immutable.
+Tuples requires all positions to have a fixed type this in turns means that the compiler knows what type each position is.
+The types used in a tuple can be different, but the types must be known at compile time.
+
+## Creating a Tuple
+
+Depending on if the tuples values types can be interprited under compilation, the tuple can be created in different ways.
+It is also important that the types of the values match the types specified in the tuple and that the number of values match the number of types specified.
+If the values are known at compile time, the tuple can be created using the tuple literal syntax.
+
+```crystal
+tuple = {1, "foo", 'c'} # Tuple(Int32, String, Char)
+```
+
+There is also possibility to create a tuple using the `Tuple` class.
+
+```crystal
+tuple = Tuple(Int32, String, Char).new(1, "foo", 'c')
+```
+
+Alternatively, you can explicitly specify the type of the variable assigned to the tuple.
+
+```crystal
+tuple : Tuple(Int32, String, Char) = {1, "foo", 'c'}
+```
+
+Explicitly specifying the type of the tuple can be usefull since that allows for defining that a position should hold a union type.
```

Comment:
```suggestion
Explicitly specifying the type of the tuple can be useful since that allows for defining that a position should hold a union type.
```